gammapy/cube/tests/test_models.py

```diff
@@ -82,21 +82,29 @@ def diffuse_evaluator(diffuse_model, exposure, background, psf, edisp):
     return MapEvaluator(diffuse_model, exposure, background, psf=psf, edisp=edisp)
 
 
+def test_BackgroundModel(background):
```

Comment:
Could you rename this to `test_background_model` to follow Python standards?

Reply:
Ok